src/views/Settings.vue

```diff
@@ -62,7 +62,8 @@
                     />
                     <amb-list-item
                       :title="version"
```

Comment:
@vickywane let's add some UX edge case handling here. If version is undefined due to some error accessing it via the API, we should show something like:
`Version information unavailable. Please make sure your edge device is online and up to date."

Reply:
I have added this use case and I also added a test case for it also.